Backport-Guide.md

```diff
@@ -26,77 +99,86 @@ into the `stable-1.2` branch of the `runtime` repo.
     $ git fetch origin
     ```
 
-    And check the list of current stable branches:
+    Check the list of stable branches:
 
     ```bash
     $ cd ${GOPATH}/src/github.com/kata-containers/runtime
     $ git branch -r | grep origin/stable
     ```
 
-- Create your branch to work on:
+- Create your branch to work on, based on the `stable-1.2` branch:
 
     ```bash
     $ git checkout -b my_1.2_pr_backport origin/stable-1.2
     ```
 
-- Locate the commits you want to pull in:
+### Locate commits to cherry pick
 
-    Here we look in the `master` branch, presuming the PR has been merged.
-    If you have the PR in a local branch you can substitute `master` for the name
-    of that branch.
+To list all commits in the `master` branch which are not in the `stable-1.2`
+branch:
 
-    ```bash
-    $ git log --oneline master
-    # And search for the SHA of the commits you wish to backport.
-    ```
+```bash
+$ git log --oneline --no-merges ..master
```

Comment:
thanks for adding the `--no-merges`. Worth making a formal note in the document body that we do *not* want merge commits cherry picked over?

Reply:
Agreed - updated.